### PR TITLE
feat(22.04): add 'copyright' slice to all SDFs

### PR DESCRIPTION
**Pull request by cjdcordeiro**
# Proposed changes

Same as #228

## Related issues/PRs

https://github.com/canonical/chisel/pull/127
### Forward porting

#228 #229 

## Checklist

* [x] I have read the [contributing guidelines](
https://github.com/canonical/chisel-releases/blob/main/CONTRIBUTING.md)
* [x] I have already submitted the [CLA form](
https://ubuntu.com/legal/contributors/agreement)

### CODE REVIEW

File: slices/libunwind-13.yaml [hunk 1]
```diff
@@ -1,4 +1,7 @@
 package: libunwind-13
+essential:
+  - libunwind-13_copyright
+
```
Comment (rebornplusplus) [inline slices/libunwind-13.yaml:4]: nit: It would be nice to have a newline separating `package` and `essential`. (Same in few other files.)

**Review by rebornplusplus — APPROVED** [review]
Looks good to me. The packages who have copyright files within them, do have the copyright slices added in this PR.

The files who do not have the copyright files for this release are the following:
* libasan6
* libgcc-s1
* libgomp1
* libncursesw6
* libpython3.11-stdlib
* libstdc++6

I only left a nit comment about spacing inside the slice definition files. But please feel free to ignore that -- it's a nitpick.

File: slices/libunwind-13.yaml (hunk 1 above)
Comment (cjdcordeiro) [reply slices/libunwind-13.yaml:5]: done

**Review by letFunny — APPROVED** [review]
Thanks!

**Pull request merged**